features/daily/line.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Line
+
+    Scenario: Change SIP line username
+        Given there are telephony users with infos:
+        | firstname | lastname | exten | context | protocol | with_phone |
+        | Ruffnut   | Thorson  |  1602 | default | sip      | yes        |
+        | Tuffnut   | Thorson  |  1603 | default | sip      | no         |
+        When I update lines with infos
+        | exten | context | username    |
+        | 1603  | default | newusername |
```

Comment:
I don't really see a use case to update multi lines in the same time in the clause WHEN. I prefer the first version, because it's more explicit that the identifier of the line is the exten@context

Reply:
I don't get it :(